index.html

```diff
@@ -52,6 +54,164 @@ <h1> Hey There. <br> I'm Dennis <br>
+              </span>
+            </p>
+          </div>
+          <div class="buttonGroup">
+            <div class="button">css</div>
+            <div class="button">html</div>
+            <div class="button">bootstrap</div>
+            <div class="button">Ruby</div>
+          </div>
+          <div class="bigButton">See Projects</div>
+        </div>
+        
+        <section id="seeProjects">
+          <div id="cardWork">
+            <div class="bg">
+              <div class="prjectDescription">
+                <span>Profesional Art <br/> Printing Data</span>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <div class="projectBtnGroup">
+                  <div class="projectBtn">html</div>
+                  <div class="projectBtn">bootstrap</div>
+                  <div class="projectBtn">Ruby</div>
+                </div>
+              </div>
+              <span class="actionBtn">See Project</span>
+            </div>
+          </div>
+
+          <div id="cardWork">
+            <div class="bg">
+              <div class="prjectDescription">
+                <span>Profesional Art <br/> Printing Data</span>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <div class="projectBtnGroup">
+                  <div class="projectBtn">html</div>
+                  <div class="projectBtn">bootstrap</div>
+                  <div class="projectBtn">Ruby</div>
+                </div>
+              </div>
+              <span class="actionBtn">See Project</span>
+            </div>
+          </div>
+
+          <div id="cardWork">
+            <div class="bg">
+              <div class="prjectDescription">
+                <span>Profesional Art <br/> Printing Data</span>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <div class="projectBtnGroup">
+                  <div class="projectBtn">html</div>
+                  <div class="projectBtn">bootstrap</div>
+                  <div class="projectBtn">Ruby</div>
+                </div>
+              </div>
+              <span class="actionBtn">See Project</span>
+            </div>
+          </div>
+
+          <div id="cardWork">
+            <div class="bg">
+              <div class="prjectDescription">
+                <span>Profesional Art <br/> Printing Data</span>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <div class="projectBtnGroup">
+                  <div class="projectBtn">html</div>
+                  <div class="projectBtn">bootstrap</div>
+                  <div class="projectBtn">Ruby</div>
+                </div>
```

Comment:
- [ ] For The list of project technologies (e.g., "HTML" and "CSS") Kindly I recommend you to use `ul` tag to group them, and `li` per icon for list of project technologies; it will make your HTML code semantically correct